.travis.yml

```diff
@@ -1,11 +1,10 @@
 language: python
 python:
-  - "2.6"
+  - "pypy" # Run slowest first so as not to hold up the build
   - "2.7"
-  - "3.3"
   - "3.4"
   - "3.5"
-  - "pypy"
+  - "3.6"
```

Comment:
According to the [Travis docs](https://docs.travis-ci.com/user/languages/python/#Specifying-Python-versions), the following are also supported.

  - "3.5-dev" # 3.5 development branch
  - "3.6-dev" # 3.6 development branch
  - "3.7-dev" # 3.7 development branch
  - "nightly"
  - "pypy3" # Pypy3 2.4.0
  - "pypy-5.3.1"

I would suggest adding all of them to be prepared for upcoming versions.

Reply:
Good idea, see https://snarky.ca/how-to-use-your-project-travis-to-help-test-python-itself/

I've added all except "pypy-5.3.1", as "pypy" is actually now the newer 5.8.0 (the docs are usually a bit out of date; "pypy3" is also 5.8.0).

"3.7-dev" and "nightly" are failing pylint (lots of `Missing module docstring (missing-docstring)`), so they can be `allow_failures` for now.